docs/plugins/syntax-trailing-function-commas.md

```diff
@@ -9,6 +9,85 @@ package: babel-plugin-syntax-trailing-function-commas
 
 This plugin allows Babel to parse [trailing function commas](https://github.com/jeffmo/es-trailing-function-commas).
 
+```js
+function clownPuppiesEverywhere(
```

Comment:
Might not be too useful but maybe we can add a repl link here just to be consistent?